community/kernel/src/docs/ops/cache.asciidoc

```diff
@@ -70,7 +47,9 @@ When configuring the amount of memory allowed for the file buffers and the JVM h
 It is important to configure the memory usage, such that the Neo4j JVM process won't need to use any swap memory, as this will cause a significant drag on the performance of the system.
 
 When reading the configuration parameters on startup Neo4j will automatically configure the parameters that are not specified.
-The cache sizes will be configured based on the available memory on the computer.
+The cache sizes will be configured based on the available memory on the computer, assuming the machine is dedicated to running Neo4j.
```

Comment:
This sounds better to me:

The cache sizes will be configured based on the available memory on the computer, with the assumption that the machine is dedicated to running Neo4j.